ieee/.htaccess

```
@@ -80,6 +80,36 @@ RewriteRule ^ieee-1752-schema/physical-activity\.json$ https://opensource.ieee.o
 ## link to only ^ieee-1752-schema or ^ieee-1752-schema/
 RewriteRule ^ieee-1752-schema(/)?$ https://opensource.ieee.org/omh/1752/ [R=302,L]
 
+##
+## ^ieee-1451.0-schema
+##
+RewriteRule ^ieee-1451.0-schema/CalibrationIdentificationTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/CalibrationIdentificationTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/CommandsTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/CommandsTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/GeographicLocationTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/GeographicLocationTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/LocationAndTitleTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/LocationAndTitleTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/MetaIdentificationTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/MetaIdentificationTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/SecurityTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/SecurityTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/SmartTransducerDataModel.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/SmartTransducerDataModel.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TextTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TextTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TimeSyncTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TimeSyncTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TransducerChannelIdentificationTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TransducerChannelIdentificationTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/UnitsExtensionTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/UnitsExtensionTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/examples/E2.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/examples/E2.xml [R=302,L]
```

If these patterns will stay the same, all these main rules and examples rules could be replaced by a single pattern with optional `examples` dir:
```suggestion
RewriteRule ^ieee-1451.0-schema/((examples/)?\w+).xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/$1.xsd [R=302,L]
```